src/core/server/rendering/rendering_service.tsx

```diff
@@ -338,12 +341,14 @@ export class RenderingService {
    * @returns {boolean} indicate if user input title is valid/invalid
    */
   public isTitleValid = (title: string, configName?: string): boolean => {
-    if (!title || title.length > 36) {
+    if (!title) {
+      return false;
+    }
+    if (title.length > 36) {
       this.logger
         .get('branding')
-        .info(
-          configName +
-            ' config is not found or invalid. Title length should be between 1 to 36 characters.'
+        .error(
+          `${configName} config is not found or invalid. Title length should be between 1 to 36 characters.`
```

Comment:
Since you are splitting the checks, do you think it maybe better splitting the error message as well?

Reply:
That sounds like a good idea. https://github.com/opensearch-project/OpenSearch-Dashboards/issues/895#issuecomment-975186941. Referenced it here to as a general improvement as this is just to prevent too much unneeded logging.